sw/device/sca/aes_serial/aes_serial.c

```diff
@@ -128,15 +130,11 @@ static simple_serial_result_t a2b_hex(const char *from, char *to, size_t num) {
  */
 static void print_cmd_response(const char cmd_tag, const char *data,
                                size_t data_len) {
-  static const char b2a_hex_values[16] = "0123456789ABCDEF";
-
-  // TODO: Switch to sw/device/lib/runtime/print.h
-  uart_send_char(cmd_tag);
+  base_printf("%c", cmd_tag);
   for (int i = 0; i < data_len; ++i) {
-    uart_send_char(b2a_hex_values[data[i] >> 4]);
-    uart_send_char(b2a_hex_values[data[i] & 0xF]);
+    base_printf("%2x", (uint32_t)data[i]);
```

Comment:
```suggestion
    base_printf("%2X", (uint32_t)data[i]);
```

Just to be safe because this code used upper-case originally.

Reply:
@mcy have you verified that this doesn't need to be uppercase?

Reply:
https://wiki.newae.com/SimpleSerial shows commands in lowercase. If this needed to be uppercase, it probably should have been documented in the file, since it's not at all obvious anywhere else.